tests/unit/base/trained_model_serialization_test.cc.py

```diff
@@ -35,6 +35,17 @@
     'CMulticlassLibSVM' # error C <= 0
 ]
 
+# Algorithms that can be prematurely stopped
```

Comment:
this is list is limited by 2 things 
1. The algorithms that can be systematically tested for serialization
2. The algorithms that do not converge before starting second iteration.

For now I have also not included any KernelMachine algorithms here